lib/src/Base/Stat/LatentVariableModel.cxx

```diff
@@ -0,0 +1,299 @@
+//                                               -*- C++ -*-
+/**
+ *
+ *  Copyright 2005-2023 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/LatentVariableModel.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Exception.hxx"
+#include "openturns/SpecFunc.hxx"
+#include "openturns/Polygon.hxx"
+#include "openturns/Text.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(LatentVariableModel)
+
+static const Factory<LatentVariableModel> Factory_LatentVariableModel;
+
+/** Parameters constructor */
+LatentVariableModel::LatentVariableModel(const UnsignedInteger nLevels,
+										 const UnsignedInteger latentDim)
```

Comment:
The tabulations are an unstable way to indent the code: please use blank spaces.
```suggestion
LatentVariableModel::LatentVariableModel(const UnsignedInteger nLevels,
                                         const UnsignedInteger latentDim)
```